.github/dependabot.yml

```diff
@@ -8,3 +8,10 @@ updates:
       - "dependency"
     # Always increase the version requirement to match the new version.
     versioning-strategy: increase
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - "github-actions"
```

Comment:
Dependabot will label GitHub Actions PRs with the `github-actions` label.
That way you can see by the label that it's different from a normal `dependency` update.